.prettierrc

```diff
@@ -1,5 +1,9 @@
 {
-  "singleQuote": true,
-  "semi": false,
```

Comment:
/ True

Reply:
- [Me]: xDai, Meme

Reply:
> / True

<!-- while uploading VID_1714445350045.mp4 --> Generate an API

Reply:
Idk what happened here but it shut down